msg/ro_RO.utf8/cubrid.msg

```diff
@@ -1267,22 +1267,22 @@ $ LOADDB
 1198 Trebuie furnizat un obiect de tipul json.
 1199 Json-ul furnizat a fost invalidat de catre schema json (Calea catre schema invalidă: %1$s, Cuvantul cheie: %2$s, Calea catre json-ul invalid furnizat: %3$s)
 1200 Cale de JSON invalida
-1201 Numele unei valori dintr-un JSON Object nu poate fi NULL.
-1202 Calea "%1$s" nu exista in %2$s
-1203 Tipul JSON-ului de la calea %1$s este invalid. Expected %2$s, dar am gasit %3$s.
+1201 NULL nu poate fi folosit pentru numele unei chei dintr-un JSON de tip Obiect.
+1202 Calea "%1$s" nu exista in %2$s.
+1203 Tipul JSON-ului de la calea %1$s este invalid. Se așteaptă %2$s, dar s-a găsit %3$s.
 1204 Cheia "%1$s" din json este duplicata
-1205 Invalid type %1$s. Expecting JSON document compatible data type (STRING or JSON).
-1206 Calea Json "%1$s" nu adresează un element al unui Json ARRAY.
-1207 Cale de JSON invalida. Index is larger than allowed by system parameter json_max_array_idx.
-1208 Invalid one/all argument. The one/all argument may only take one of these two values: "one", "all".
+1205 Tipul %1$s este invalid. Se așteaptă un tip compatibil cu un document JSON (STRING sau JSON).
+1206 Calea Json "%1$s" nu adresează un element al unui Json de tip Vector.
+1207 Cale de JSON invalida. Indexul este mai mare decât valoarea permisă de parametrul de sistem "json_max_array_idx".
```

Comment:
Should we put the name of system parameter as an argument to the message catalog ?

Reply:
We could do that, yes.